tests/x11/awesome_menu.pm

```diff
@@ -0,0 +1,15 @@
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    send_key "super-w";
+	sleep 1;
+    mouse_hide(1);
+	sleep 1;
```

Comment:
travis checks fail here. You can (and should) execute tests locally before submitting. Call
`make test`
in your working copy to call the tests.